fe/fe-core/src/main/java/com/starrocks/connector/elasticsearch/external/EsMetaStateTracker.java

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package com.starrocks.external.elasticsearch;
+package com.starrocks.connector.elasticsearch.external;
```

Comment:
I think external dir could be removed. Just put these files under elasticsearch dir.